core/contracts/common/implementation/MultiRole.sol

```diff
@@ -34,17 +34,17 @@ library Shared {
         return roleMembership.members[memberToCheck];
     }
 
-    function addMember(RoleMembership storage roleMembership, address memberToAdd) internal {
+    function addSharedMember(RoleMembership storage roleMembership, address memberToAdd) internal {
         roleMembership.members[memberToAdd] = true;
     }
 
-    function removeMember(RoleMembership storage roleMembership, address memberToRemove) internal {
+    function removeSharedMember(RoleMembership storage roleMembership, address memberToRemove) internal {
```

Comment:
Should we get rid of all the multirole changes given that we don't want the interface to change? (I know these are internal changes, but I think they actually wanted us to change the external function names)

Reply:
ye, I've reverted all changes to multirole re our discussion. Thinks keeps things simple. no need to change more